docs/userguide/datetime/timezone/index.md

```diff
@@ -141,47 +136,47 @@ The ICU resource files required for time zone data updates are posted at
 <https://github.com/unicode-org/icu-data/tree/master/tzdata/icunew>. The
 required resource files for ICU version 44 and newer are
 
-*   zoneinfo64.res
-*   windowsZones.res
-*   timezoneTypes.res
-*   metaZones.res
+* zoneinfo64.res
+* windowsZones.res
+* timezoneTypes.res
+* metaZones.res
 
-#### ICU4C TZ update of a .dat Package File
+### ICU4C TZ update of a .dat Package File
 
 For ICU configurations that load data from a .dat package file, replace the time
 zone resources in that file.
 
-1.  Download the new .res files from
-    http://source.icu-project.org/repos/icu/data/trunk/tzdata/icunew/<*iana tz
-    version number*>/44/<*platform directory>/.*
-    *   *"44" *is the directory for updates to ICU version 4.4 and newer.
-    *   <*platform directory>* is "le" for little endian processors, including
-        all Intel processors.
-    *   <*platform directory>* is "be" for big endian processors, including IBM
-        Power and Sparc.
-    *   <*platform directory>* is "ee" for IBM mainframes using EBCDIC character
-        sets.
-2.  Check that the tool "icupkg" is available. If not already on your system,
-    you can get it by [downloading](http://site.icu-project.org/download) and
-    building ICU, following the instructions in the ReadMe file included in the
-    download. Alternatively, on many Linux systems, "apt-get install
-    icu-devtools" will install the tool.
-3.  Locate the .dat file to be updated, and do the update. The commands below
-    are for a .dat file named icudt55l.dat.
-
-` icupkg -a zoneinfo64.res icudt55l.dat`
-
-` icupkg -a windowsZones.res icudt55l.dat`
-
-` icupkg -a timezoneTypes.res icudt55l.dat`
-
-` icupkg -a metaZones.res icudt55l.dat`
+1. Download the new .res files from
+   `http://source.icu-project.org/repos/icu/data/trunk/tzdata/icunew/`
+   `<IANA tz version>/44/<platform directory>`.
+   * `<IANA tz version>` is a combination of year and letter, such as "2019c".
+   * *"44"* is the directory for updates to ICU version 4.4 and newer.
+   * `<platform directory>` is "le" for little endian processors, including
+     all Intel processors.
+   * `<platform directory>` is "be" for big endian processors, including IBM
+     Power and Sparc.
+   * `<platform directory>` is "ee" for IBM mainframes using EBCDIC character
+     sets.
+1. Check that the tool "icupkg" is available. If not already on your system,
+   you can get it by [downloading](https://github.com/unicode-org/icu/releases)
+   and building ICU, following the instructions in the ReadMe file included in
+   the download. Alternatively, on many Linux systems, "apt-get install
+   icu-devtools" will install the tool.
+1. Locate the .dat file to be updated, and do the update. The commands below
```

Comment:
FYI / optional

This auto-numbering works, but I tend to manually number the bullets so that the .md is more readable.

Reply:
Updated numbered list to use sequential numbers.